.travis.yml

```diff
@@ -27,10 +30,9 @@ jobs:
         - bash <(curl -s https://codecov.io/bash)
 
     - name: "Integration Tests"
-      cache: false
       install:
         - pip install tox-travis coverage
-        - pushd .. && git clone --single-branch --branch master https://github.com/lbryio/torba.git && popd
+        - pip install git+https://github.com/lbryio/torba.git@${TORBA}#egg=torba
```

Comment:
this doesn't make sense, why did you change it from checking out the correct version to install with pip?

Reply:
it installs the same version by default - `master` - determined by the `TORBA` travis variable

Reply:
install it into what?